modules/using-aws-cli-create-iam-role-alb-controller.adoc

```diff
@@ -4,13 +4,13 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="using-aws-cli-create-iam-role-alb-controller_{context}"]
-= Using the AWS CLI to create an IAM role for the Controller
+= Creating an AWS IAM role for the AWS Load Balancer Controller
```

Comment:
If change to this, then we have two same subtitle of `Creating an AWS IAM role for the AWS Load Balancer Controller`

Reply:
Addressed

Reply:
Modified the title